specs/architecture.md

```diff
@@ -9,16 +9,16 @@ Architecture
 | name              | description                                                                                                                                                                      |
 | ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | App (application) | Alternate name for "[virtual sidechain](https://arxiv.org/abs/1905.09274)." LazyLedger apps are sidechains that post all their data onto the LazyLedger chain to share security. |
-| Transaction       | Requests that modify the consensus-critical state (validator balances and statuses).                                                                                             |
-| Message           | Requests that are executed by non-consensus-critical apps.                                                                                                                       |
+| Transaction       | Request that modifies the consensus-critical state (validator balances and statuses).                                                                                            |
+| Message           | Request that is executed by a non-consensus-critical app.                                                                                                                        |
```

Comment:
Didn't we want to move away from the word Request here?

Reply:
I don't think so? It's a term taken from the Tendermint paper. If there's an alternative for "atomic blob that is processed by the state machine," then I can change it.

Reply:
> "atomic blob that is processed by the state machine," 

Transaction?

Reply:
Okay, well, it's outside the scope of this PR. I'll file an issue to follow up on potentially better nomenclature for this. #101